content_entity_base.module

```diff
@@ -20,5 +20,42 @@ function content_entity_base_help($route_name, RouteMatchInterface $route_match)
       return $output;
 
     default:
+      return '';
+  }
+}
+
+/**
+ * Implements hook_theme_registry_alter.
+ */
+function content_entity_base_theme_registry_alter(&$theme_registry) {
+  // Check for entity_add_list which was added in 8.1.x.
+  if (!isset($theme_registry['entity_add_list'])) {
```

Comment:
I have to deal with some installs that have yet to upgrade to 8.1.x. =( 

I doubt I am the only one.